rapids-cmake/cython/set_lib_dirs.cmake

```diff
@@ -0,0 +1,53 @@
+# =============================================================================
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+# =============================================================================
+
+include_guard(GLOBAL)
+
+#[=======================================================================[.rst:
+rapids_cython_set_lib_dirs
+--------------------------
+
+.. versionadded:: v22.12.00
+
+Set the directories where any dependent libraries will be placed.
```

Comment:
The short description should outline we will populate the ``RAPIDS_CYTHON_<ProjectName>_LIB_DIRS` variable